lib/application/constants/strings.dart

```diff
@@ -16,18 +16,51 @@ const collectionsExploreTab = 'Explorar';
 const collectionsReviewTab = 'Revisar';
 
 const collectionsSectionHeaderSeeAll = 'Ver todos';
-const collectionsMemoryStability = 'Estabilidade da Memória';
-String collectionsCompletionProgress({required int current, required int target}) =>
-    '$current / $target memos completados';
 
-const progressTotalStudyTime = 'Horas totais de estudos';
+//
+// Progress
+//
+const progressTotalStudyTime = 'Totais de estudos';
```

Comment:
It shouldn't be `estudo` instead `estudos`? I'm not sure

Reply:
I think both are correct